analyses/tcd/metrics/TCD_trigger_report.Rmd

```diff
@@ -0,0 +1,432 @@
+                                     targeted_window_b, 
+                                     lead_time_b, 
+                                     data_source_b,
+                                     trigger_b,
+                                     hist_period_b,
+                                     monitored_by_b,
+                                     package_b, 
+                                     paste0(prob_b, "%, based on global IRI data, but very high uncertainty"), 
+                                     funding_b_formatted,
+                                     expected_cost_b,
+                                     "N/A",
+                                     "N/A"),
+                       trigger_c = c(type_c, 
+                                     monitored_area_c,
+                                     timepoints_c, 
+                                     targeted_window_c, 
+                                     lead_time_c, 
+                                     data_source_c,
+                                     trigger_c,
+                                     hist_period_c,
+                                     monitored_by_c,
+                                     package_c, 
+                                    "<img src=plots/trimmed_Trigger3_atv.png>", 
+                                     funding_c_formatted,
+                                     expected_cost_c,
+                                     "<img src=plots/trimmed_Trigger3_var_tradeoff_bar.png>",
+                                     "<img src=plots/trimmed_Trigger3_det_tradeoff_bar.png>"))
+
+kbl(snapshot_df,
+    format = 'html',
+    escape = FALSE,
+    row.names = FALSE, # omit row numbers
+    col.names = c("","Trigger 1", "Trigger 2", "Trigger 3"), 
+    align = "rccc") %>%
+  kable_minimal(c("hover", "striped")) %>%
+  row_spec(1:15, extra_css = "line-height: 30px;") %>%
+  column_spec(column = 1, extra_css = "vertical-align: middle;") %>%
+  column_spec(column = c(2:4), width = "28em") %>%
+  pack_rows("Description", start_row = 1, end_row = 8, color = '#1bb580') %>%
+  pack_rows("Programming and Funding", start_row = 9, end_row = 12, color = '#1bb580') %>%
+  pack_rows("Performance", start_row = 14, end_row = 15
+            , color = '#1bb580') %>%
+  kableExtra::footnote(general_title = "Notes",
+                       general = trigger_table_footnote)
+
+```
+
+&nbsp;
+
+## Activation Scenarios
+<p style="color:black;font-size:16px;text-align:left;"> Due to the lack of historical data for the IRI seasonal forecast, we do not assess the likelihood of activation scenarios together. Refer to the likelihood of the individual triggers activating.</p>
+
+&nbsp;
+
+## Technical Team
+<p style="color:black;font-size:16px;text-align:left;"> `r data_providers` have provided data and/or analytical support. A total of `r nbr_hist_shocks` historical occurrences of severe shocks were documented and used to develop the trigger mechanism. `r analysis` </p>
```

Comment:
There were 5 historical shocks / years documented, but only 5 or so years of forecast to work with. how many shocks occurred on years for which we had forecasts? What time span do we have historical shock data for?  

Important to be clear that we could only set a threshold and estimate performance based on 5 years of data for which we had forecasts + impact data.